### DISCUSSION TITLE
Error when exporting source code

### DISCUSSION THREAD

**Discussion by StefanB1983** [joyfullservice/msaccess-vcs-addin#309] [Q&A] [answered]
Hi,
I have installed the Version Control System. Looking great and very user-friendly. 
When I export the source code I face on error though about a query that can not be exported. It is a big query with > 1000 lines. Do you know why the error occurs and how I can resolve this?
Kind regards,
Stefan

**Comment by joyfullservice** [comment, marked as answer]
Hi @StefanB1983 - I have moved this over to issue #310 so we can track the resolution and link any programming changes.
